src/components/Footer/Footer.js

```diff
@@ -96,7 +214,7 @@ const Footer = ({
               disabled={item?.isBlurredPost}
               style={styles.btn}
               onPress={onPressComment}>
-              <View style={styles.btnComment}>
+              <View style={styles.btnComment(totalComment || 0)}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `btnComment` style is now a function that takes `commentValue` as an argument. This allows the paddingRight to be adjusted based on the number of comments. This is a clever way to adjust the style dynamically based on the data.
```diff
- <View style={styles.btnComment(totalComment || 0)}>
+ <View style={styles.btnComment}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->